scripts/cleanup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Please run this script only in the project root folder.
+#
+# Usage
+# ./scripts/cleanup.sh
```

Comment:
Ideally I'd like a `npm run clean` script in each package, and a `"clean": "lerna run clean"` script in the root.